src/styles/blocks/catalog.scss

```diff
@@ -11,12 +10,18 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
-
-    width: 200px;
-    height: 300px;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $transition-duration ease-in-out;
```

Comment:
one time my work was rejected cuz of "all" in transition, mentor said that u need to write every property

Reply:
Oh, then in the future I will do so. Fixed